src/plugins/plugin.filler/index.js

```diff
@@ -53,7 +54,7 @@ export default {
     const area = chart.chartArea;
     for (let i = metasets.length - 1; i >= 0; --i) {
       const source = metasets[i].$filler;
-      if (!source) {
+      if (!_shouldApplyFill(source)) {
```

Comment:
I think the culprit is here, the following `updateControlPoints` needs to be called for all lines (source is defined) to make the control points available on the non-filled lines as fill target.